fmriprep/workflows/bold/registration.py

```diff
@@ -32,6 +33,57 @@
 
 DEFAULT_MEMORY_MIN_GB = 0.01
 
+class PatchedBBRegisterRPT(BBRegisterRPT):
+    def _list_outputs(self):
```

Comment:
I would prefer to move this to `_post_run_hook`, rather than `_list_outputs`. (I know that `PatchedConcatenateLTA` has it there. We should change that.)

Please see https://github.com/poldracklab/fmriprep/issues/1268#issuecomment-419446891 for the general structure of a `_post_run_hook`.